source/RoslynAnalyzers/MessageContractAnalyzers_Conventions.cs

```diff
@@ -32,11 +31,42 @@ static bool ApiContractTypes_MustLiveInTheAppropriateNamespace(SyntaxNodeAnalysi
             return false;
         }
 
+        static bool WeDoNotUseEnums_InOurApiSurface(SyntaxNodeAnalysisContext context, PropertyDeclarationSyntax propDec, bool isExperimental)
+        {
+            // types marked experimental such as BFF's are allowed to use enums as they aren't constrained by compatibility across more than one version.
+            if (isExperimental) return true;
+            
+            var systemEnum = context.Compilation.GetSpecialType(SpecialType.System_Enum);
+
+            var propType = propDec.Type is NullableTypeSyntax n ? n.ElementType : propDec.Type;
+            var t = context.SemanticModel.GetTypeInfo(propType);
```

Comment:
Nit: Abbreviations. I propType is a bit short for my taste, but I feel "t" as a name should be expanded to provide more meaning.